onnx/helper.py

```diff
@@ -39,27 +39,38 @@ def make_graph(nodes, name, inputs, outputs, initializer=[]):
     graph.initializer.extend(initializer)
     return graph
 
-def make_tensor(name, data_type, dims, vals):
+
+def make_tensor(name, data_type, dims, vals, raw=False):
```

Comment:
Perhaps now is a good time to add a proper docblock here ;)

Reply:
Especially because `raw` changes the expected type of what you pass for `vals`.

Speaking of which, we don't have a utility function for taking a numpy ndarray and making a TensorProto from it. Might be useful.

Reply:
Sure will add a docstring.

There is one in numpy_helper https://github.com/onnx/onnx/blob/master/onnx/numpy_helper.py#L94 .